common/core/src/authorization.ts

```diff
@@ -39,21 +39,64 @@ export function hmacHash(password: string, stringToSign: string): string {
   return hmac.digest('base64');
 }
 
```

Comment:
The previous 15 lines are a weird color in codeflow.  Don't really see what's causing that.

Reply:
looks like it's the regex on line 24

---
In reply to: [157662646](https://github.com/Azure/azure-iot-sdk-node/pull/184#discussion_r157662646) [](ancestors = 157662646)